lib/finder_frontend.rb

```diff
@@ -93,10 +92,8 @@ def filter_params
         }
     end
 
-    def document_type_param
-      {
-        "filter_document_type" => document_type,
-      }
+    def default_filter
+      finder.filter.to_h
```

Comment:
@evilstreak About your point the other day, that `filter_` is prepended to the `filter_params` - should I modify this to do that here too and remove `filter_` from the content item, or should we be explicit there and let it do it's thing here?